internal/distro/rhel7/images.go

```diff
@@ -129,15 +129,6 @@ func osCustomizations(
 		osc.SELinuxForceRelabel = imageConfig.SELinuxForceRelabel
 	}
 
-	if oscapConfig := c.GetOpenSCAP(); oscapConfig != nil {
-		osc.OpenSCAPConfig = osbuild.NewOscapRemediationStageOptions(
-			osbuild.OscapConfig{
-				Datastream: oscapConfig.DataStream,
-				ProfileID:  oscapConfig.ProfileID,
-			},
-		)
-	}
-
```

Comment:
why was this deleted?

Reply:
Because it's essentially dead code. We validate this earlier and return and error if a user tries to provide oscap customizations for a RHEL7 image. I can leave it in.

Reply:
I'm inclining to keep `images.go` as similar as possible across various distros, since I see them as candidates for deduplication. But removing this is not a big deal. I'll leave it up to you, as I'm OK with either way after your explanation.

Reply:
I left it in the end :)